webapp/src/Service/DOMJudgeService.php

```diff
@@ -53,6 +53,7 @@
 use Symfony\Component\Security\Core\Authentication\Token\UsernamePasswordToken;
 use Symfony\Component\Security\Core\Authorization\AuthorizationCheckerInterface;
 use ZipArchive;
+use function count;
```

Comment:
Do we need this?

Reply:
No, it was a mixup in my phpstorm config. Fixed.